content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -129,6 +129,109 @@ kubectl delete jobs/job-pod-failure-policy-ignore
 
 The cluster automatically cleans up the Pods.
 
+## Using Pod failure policy to avoid unnecessary Pod retries based on custom Pod Conditions
+
+With the following example, you can learn how to use Pod failure policy to
+avoid unnecessary Pod restarts based on custom Pod Conditions.
+
+{{< note >}}
+The example below works since version 1.27 as it relies on transitioning of
```

Comment:
mention the FeatureGate

Reply:
Done, but added the note into the "Before you begin" section at the top, as this is not specific to this example.